docs/user/guides/custom-domains.rst

```diff
@@ -0,0 +1,121 @@
+How To Manage Custom Domains
+============================
+
+This guide describes how to host your documentation using your own domain name, such as ``docs.example.com``.
+
+.. contents:: Contents
+    :local:
+
+.. _adding_domain:
```

Comment:
IIRC, these are not needed since we use `autosectionlabel` extension: https://www.sphinx-doc.org/en/master/usage/extensions/autosectionlabel.html

Reply:
Yes, this must be some artifact from a refactor where I thought I had a wrong incoming reference. It doesn't even seem to be referenced. Removed! Thanks!